app/connector/rest-swagger/src/main/resources/META-INF/syndesis/connector/rest-swagger.json

```diff
@@ -13,7 +13,7 @@
   "connectorFactory": "io.syndesis.connector.rest.swagger.ConnectorFactory",
   "dependencies": [
     {
-      "id": "@project.groupId@:@project.artifactId@:@project.version@",
+      "id": "io.syndesis.connector:connector-rest-swagger:@project.version@",
```

Comment:
hmm odd, works without issues in my IDE, is this a permanent change? Doesn't the `@project.version@` influence the running of tests as well?

Reply:
It should only work if you have run a Maven build before so Maven already applied the filtering in build output directory. When you do `mvn clean` from time to time you should get the errors.

I think @project.version@ makes sense and the version is not checked in unit tests.

Reply:
I think my IDE is running Maven filtering on resources.

Reply:
Lucky you 😃